hermes_core/__init__.py

```diff
@@ -20,4 +20,6 @@
 MISSION_NAME = "hermes"
 INST_NAMES = ["eea", "nemisis", "merit", "spani"]
 INST_SHORTNAMES = ["eea", "nms", "mrt", "spn"]
+INST_TARGETNAMES = ["EEA", "MAG", "MRT", "SPANI"]
```

Comment:
```suggestion
INST_TARGETNAMES = ["EEA", "MAG", "MERIT", "SPANI"]
```

On testing with the l0 files synced from the MOC, the pneumonic for Merit is actually `MERIT` instead of `MRT` (Ex. `hermes_MERIT_l0_2022260-030002_v01.bin`)